pinot-common/src/test/java/org/apache/pinot/common/utils/PinotDataTypeTest.java

```diff
@@ -105,6 +105,7 @@ public void testBytes() {
   @Test
   public void testTimestamp() {
     Timestamp timestamp = Timestamp.valueOf(LocalDateTime.now());
+    timestamp.setNanos(0);
```

Comment:
Is this related?

Reply:
yes, java11 uses nano by default, Pinot converter doesn't keep the nano part.

Reply:
In that case, maybe cleaner if directly initialize with millis
```suggestion
    Timestamp timestamp = new Timestamp(System.currentTimeMillis());
```